torch/csrc/jit/tensorexpr/loopnest.h

```diff
@@ -238,6 +238,9 @@ class TORCH_API LoopNest {
   // S6:    for k
   // S7:      B[i] = B[i] +
   static std::vector<For*> distributeLoop(For* loop);
+  // Same as above, but also distribute parent loops.
+  // Returns the result of distributing the outermost loop.
+  static std::vector<For*> distributeLoopAndParents(For* loop);
```

Comment:
Could you please add an example of what it does? I have hard times understanding what exactly "also distribute parent loops" means.

Reply:
Added an example for these 2 APIs as well.